library/Makefile

```diff
@@ -356,6 +356,8 @@ $(TF_PSA_CRYPTO_GENERATED_CONFIG_CHECK_FILES):
 	echo "  Gen   $(TF_PSA_CRYPTO_GENERATED_CONFIG_CHECK_FILES)"
 	$(PYTHON) $(TF_PSA_CRYPTO_CORE_PATH)/../scripts/generate_config_checks.py
 
+$(TF_PSA_CRYPTO_CORE_PATH)/tf_psa_crypto_config.o: $(TF_PSA_CRYPTO_GENERATED_CONFIG_CHECK_FILES)
```

Comment:
That part won't be necessary to pass the PR CI once [Windows-mingw job is removed](https://github.com/Mbed-TLS/mbedtls/pull/10382). It will still be necessary so that `make lib` works from a fresh checkout, which is nice even if the CI doesn't require it.